src/reader/token.h

```diff
@@ -1,5 +1,26 @@
+#include <string_view>
 #pragma once
 
 class Token {};
 
-class LeftParenthesis : Token {};
+class LeftParenthesis : public Token {};
+class RightParenthesis : public Token {};
+
+class Identifier : public Token {
+    std::string_view value;
+};
+
+class Integer : public Token {
+    int value;
```

Comment:
```suggestion
    int64_t value;
```

с этим надо бы более точным быть